tamr_client/__init__.py

```diff
@@ -56,3 +56,9 @@
     AttributeNotFound,
 )
 import tamr_client.attributes.attribute as attribute
+
+# records
+import tamr_client.datasets.record as record
```

Comment:
You could consider additionally `from tamr_client.datasets.record import PrimaryKeyNotFound` so that its available at the top-level as `tc.PrimaryKeyNotFound`. We already do this for other exception types in `tamr_client` above.

Reply:
Done. 👍

Reply:
Agreed. Done.